yt/admin/dashboards/yt_dashboards/bin/main.py

```diff
@@ -54,10 +65,21 @@
             "args": [False],
         },
     },
+    # Supported backends: monitoring, grafana.
     "scheduler-pool": {
         "func": build_scheduler_pool,
         "monitoring": {},
-        "grafana": {},
+        "grafana": {
+            "id": "<your-dashboard-id>",
+            "postprocessor": combine_postprocessors(
```

Comment:
It may be reasonable to move these postprocessors out of the list and make a generic function since there is almost no specifics to scheduler. This generic function may take value_mapping or so.

Reply:
I assumed that this list in `main.py` is an example to follow, rather than the final configuration one should use for their own installation. That is why I only provided one example with dummy parameters and extra comments, since the configuration would obviously differ for each particular use (and the actual dashboards-as-code should be in a separate file).

In an actual configuration this whole part will be about the same length as a function call with `extra_tags` and `value_mapping` as arguments, IMO, so it is fine to repeat it (probably storing these tags/mappings in a global variable in the beginning of the file). 
For this reason I don't see much use in a function.

But maybe you have an example/rationale that will change my mind.